### PR TITLE
feat: lint GitHub Actions workflows TDE-919

### PR DESCRIPTION
**Pull request by l0b0**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -54,4 +54,5 @@ pytest = "^7.4.3"
 pytest-dependency = "^0.5.1"
 moto = "^4.2.6"
 pytest-mock = "^3.12.0"
+shellcheck-py = "*"
```
Comment (paulfouquet) [inline pyproject.toml:57]: Where is that used? Is that something you run manually?

Comment (l0b0) [reply pyproject.toml:57]: It's an optional dependency of actionlint.

Comment (paulfouquet) [reply pyproject.toml:57]: How does that work? `actionlint` detects automatically that this dependency exists and run it?

Comment (l0b0) [reply pyproject.toml:57]: Exactly.

**Review by paulfouquet — APPROVED** [review]

**Pull request merged**